### ISSUE TITLE
Nonsensical Method

### ISSUE THREAD

**Issue by psygate** [CivMC/SimpleAdminHacks#26]
* Code Location:
https://github.com/CivMC/SimpleAdminHacks/blob/927ba898f2bf8cf65edb2550b946a8375b0f2293/paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/SimpleAdminHacks.java#L68

* Issue:
getClassLoader is guaranteed by the java standard to exist on class objects. [See here](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Class.html#getClassLoader()), the only case where this is null is the bootstrap classloader, which can't be accessed by userspace code easily anyways. This method does nothing but hide the well known access location of getClassLoader.

* Solution:
Remove this method.

**Comment by Protonull** [comment]
Yeah, I thought it was weird when I saw it in my "[big scary](https://github.com/CivMC/SimpleAdminHacks/commit/f26710bb38cc2d14f035f0137737df0d55a9778a)" commit, but I figured it must've been [added](https://github.com/CivMC/SimpleAdminHacks/commit/9861b67ab1cff73b8dd787178ac3eaed6b70828d#diff-76fc2bd4e768143b7d997cdf3bd6e6fdf28d4dc271ff41887247381359168756R338-R340) for weird Java 8 reasons or some such so I didn't touch it except to move it from the bottom of the class. I nonetheless agree that it can be replaced with with `SimpleAdminHacks.class.getClassLoader()`.
